### PR TITLE
Fixed bug with scrubbables producer interface

### PR DESCRIPTION
**Pull request by jasonhowmans**
Fixed bug with scrubbables producer interface where timeline level links were loading as timeline segments...

### CODE REVIEW

File: atlas-admin-frontend/js/directives/scrubber.js [hunk 1]
```diff
@@ -218,7 +218,7 @@ app.directive('scrubber', ['$document', '$compile',
             label: label,
             url: url,
             startTime: startTime,
-            endTime: endTime,
+            endTime: startTime + endTime,
```
Comment (Lukewh) [inline atlas-admin-frontend/js/directives/scrubber.js]: If endTime is actually a duration, maybe rename the var? If not, why is endTime startTime + endTime? Maybe a little comment explaining? xox

File: atlas-admin-frontend/js/dist/scripts.js [hunk 2]
```diff
@@ -10849,6 +10855,7 @@ function($scope, $rootScope, $routeParams, $q, Scrubbables, $timeout, Helpers) {
       $scope.showUI = true;
       $scope.loading = false;
       pushSegmentsToTimeline();
+      console.log($scope.episode);
```
Comment (Lukewh) [inline atlas-admin-frontend/js/dist/scripts.js]: After removing all dem console.logs :(

**Comment by Lukewh** [comment]
:+1:

File: atlas-admin-frontend/js/dist/scripts.js (hunk 2 above)
Comment (yemble) [inline atlas-admin-frontend/js/dist/scripts.js]: loglevellllllllll js

**Pull request merged**